### DISCUSSION TITLE
tfly messages adjustments more in depth

### DISCUSSION THREAD

**Discussion by Truongvv10** [Zrips/CMI#7224] [Feature Requests]
An option to make tfly display _```messages in different ways```_, like:
- Actionbar
- Bossbar
- Chat
- Title
- Goal

An option to have tfly _```messages disabled```_ would be nice so we can stop the spam messages it gets in actionbar. This gets annoying when using for example towny where actionbar messages gets displayed repeatedly where both just go back and forth flickering.

An option to have tfly _```world blacklist/white```_, so players can't fly in certain worlds.

An option to have tfly be _```silent output```_, both for players and console. Probably with __/cmi tfly Zrips +30 -s__

**Comment by mrfloris** [comment]
Thank you for moving it here. And I agree, it would be kinda nice if we can select the 'type' of output for fly, ingame chat, bossbar, or actionmsg.
